datapackage_pipelines_budgetkey/pipelines/education/tochniyot/scraper.py

```diff
@@ -0,0 +1,140 @@
+    :return: the education programs data
+    """
+    education_programs_count = get_education_programs_count()
+    json_data = send_tochniyot_request(education_programs_count)
+    tochniyot_data = json_data['d']['Data']  # returns an array of all the education programs
+    return tochniyot_data
+
+
+def send_tochniyot_request(records_num=1):
+    """
+    Sends tochniyot_url to get education programs data
+    :param records_num: the number of tochniyot to fetch.
+    :return: the json data in the response. Contains the programs data and
+    some metadata about it (e.g. total number of programs)
+    """
+    tochniyot_search_params[records_count_field_name] = records_num
+    resp = requests.post(tochniyot_url, json=tochniyot_search_params)
+    if resp.status_code != 200:
+        logging.error('Failed request: could not get tochnyiot data.')
+        raise requests.exceptions.HTTPError(f'Request for Tochnyot data failed. URL: {tochniyot_url}\n'
+                                            f'content: {tochniyot_search_params}\n'
+                                            f'response code: {resp.reason}')
+    json_data = json.loads(resp.text)
+    return json_data
+
+
+def TaarichStatus_to_date(row):
+    """
+    transform a JS format date to a date parsable by dataflows
+    :param row: row of data
+    :return: a row with parsable date
+    """
+    js_date_str = row[TAARICH_STATUS]  # TaarichStatus format: /Date(<some long integer>)/
+    js_date_integer_only = int(js_date_str.split('(')[1].split(')')[0])  # extracts the integer from js_date_str
+    try:
+        row[TAARICH_STATUS] = datetime.utcfromtimestamp(js_date_integer_only / 1000).strftime(LONG_DATE_FORMAT)
+    except ValueError as e:
+        logging.error(f"TaarichStatus column is not in the expected date format.\n{e}")
+
+
+def TaarichIdkunReshuma_to_full_date(row):
+    """
+    TaarichIdkunReshuma this field is sometimes in '%d/%m/%Y %H:%M:%S' format and sometimes in '%d/%m/%Y'
+    Change all values to '%d/%m/%Y %H:%M:%S'
+    :return:
+    """
+    date_str = row[TAARICH_IDKUN_RESHUMA]
+    if short_date_regex.fullmatch(date_str) is not None:  # the date is in SHORT_DATE_FORMAT
+        try:
+            row[TAARICH_IDKUN_RESHUMA] = datetime.strptime(date_str, SHORT_DATE_FORMAT).strftime(LONG_DATE_FORMAT)
+        except ValueError as e:
+            logging.error(f"TaarichIdkunReshuma column is not in the expected date format.\n{e}")
+
+
+def flow(*_):
+    return Flow(scrape(),
+                TaarichStatus_to_date,
+                TaarichIdkunReshuma_to_full_date,
+                set_type('MisparTochnit', type='number'),
+                set_type('CodeYechidaAchrayit', type='number'),
+                set_type('CodeTchumMerkazi', type='number'),
+                set_type('MisparTaagid', type='number'),
+                set_type('KayamNatsigMisrad', type='number'),
+                set_type('TaarichStatus', type='datetime', format=LONG_DATE_FORMAT),
```

Comment:
You might try using 'any' as the format which uses python-dateutil to parse the date (instead of a fixed format).